app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 #:reek:NilCheck
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
```

Comment:
what is it?

Reply:
It is check for valid CSRF token. At the beginning of developing these models we had some problems with CSRF token, so i disable check in backend to allowed continue contribution. Now ti is no need for this.